compiler/luci-value-test/tester/src/EvalTester.cpp

```diff
@@ -141,24 +149,30 @@ int entry(int argc, char **argv)
 
   // Get output.
   const auto output_nodes = loco::output_nodes(module->graph());
-  // TODO: Support multiple outputs
-  assert(output_nodes.size() == 1);
-  const auto *output_node = dynamic_cast<const luci::CircleOutput *>(output_nodes[0]);
-  std::vector<char> output_data(getTensorSize(output_node));
-  interpreter.readOutputTensor(output_node, output_data.data(), output_data.size());
-
-  // Output data is written in ${output_file}
-  // (ex: Add.circle.output)
-  // Output shape is written in ${output_file}.shape
-  // (ex: Add.circle.output.shape)
-  // TODO: Use HDF5 file format
-  writeDataToFile(output_file, output_data.data(), output_data.size());
-  auto shape_str = std::to_string(output_node->dim(0).value());
-  for (int i = 1; i < output_node->rank(); i++)
+  for (int i = 0; i < module->graph()->outputs()->size(); i++)
   {
-    shape_str += ",";
-    shape_str += std::to_string(output_node->dim(i).value());
+    const auto *output_node = dynamic_cast<const luci::CircleOutput *>(output_nodes[i]);
+    std::vector<char> output_data(getTensorSize(output_node));
+    interpreter.readOutputTensor(output_node, output_data.data(), output_data.size());
+
+    // Output data is written in ${output_file}
+    // (ex: Add.circle.output)
+    // Output shape is written in ${output_file}.shape
+    // (ex: Add.circle.output.shape)
+    // TODO: Use HDF5 file format
+    writeDataToFile(output_file, output_data.data(), output_data.size(), i != 0);
+    auto shape_str = std::to_string(output_node->dim(0).value());
+    if (i != 0)
+    {
+      shape_str = "\n" + shape_str;
+    }
+    for (int i = 1; i < output_node->rank(); i++)
+    {
+      shape_str += ",";
+      shape_str += std::to_string(output_node->dim(i).value());
+    }
```

Comment:
1. Why do you use a different format from multiple inputs?

2. This code is a little bit confusing. Please use a different variable name (`i`).

3. It would be better to make a string in the loop and write the string to the file outside the loop. (to reduce # of file IO)

Reply:
1.
2. that's my mistake. I will fix it.
3. Okay, I will make a Change.

Reply:
1. I will change multiple output file format. 
2. that's my mistake. I will fix it.
3. Okay, I will make a Change.